__tests__/app.test.js

```diff
@@ -155,3 +155,69 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST : /api/articles/:article_id/comments", () => {
```

Comment:
Good tests but would benefit from the following additional tests:
test for endpoint ignoring unnecessary body object properties,
test for non existent ID, e.g. 0 or 9999,